quick-start.md

```diff
@@ -2,222 +2,87 @@
+You can sign-up to Kontena Cloud by following this link: https://cloud.kontena.io/sign-up
 
-You can install Kontena CLI using our [official installer](https://gh-releases.kontena.io/kontena/kontena/pkg/latest) or [Homebrew](https://brew.sh/) :
+## Step 2. Create an Organization
 
-```
-$ brew install kontena
-```
-
-### Debian / Ubuntu
-
-You can install Kontena CLI using our [official deb package](https://gh-releases.kontena.io/kontena/kontena/deb/latest).
-
-### Linux / Windows
-
-> Prerequisites: You'll need Ruby version 2.1 or later installed on your system. For more details, see the official [Ruby installation docs](https://www.ruby-lang.org/en/documentation/installation/).
-
-You can install the Kontena CLI using the Rubygems package manager (which is included in Ruby).
-
-```
-$ gem install kontena-cli
-```
+After you have signed up and verified your email, you can create an organization for your project. An organization is a workspace for development teams to monitor, manage and operate Kontena Platforms.
 
-After the installation is complete, you can test the installation by checking the Kontena CLI version with `kontena version`.
+<img src="_images/kontena-cloud-create-org.png" alt="Create Organization" />
 
-## Step 2. Create a Kontena Platform
+## Step 3. Create a Kontena Platform
 
-The easiest (and preferred) way to provision Kontena Platform Master is to use the built-in Kontena Cloud Platform provision feature of Kontena CLI. In this guide, we will provision Kontena Platform to Kontena Cloud and nodes to the local development environment using [Vagrant](https://www.vagrantup.com/). If you want to install nodes to some other environment, please see [Installing Kontena](using-kontena/install-nodes/) documentation.
+Kontena Platform will abstract all available compute resources and make these resources available to your container workloads. To create your first Kontena Platform, just open the 'Create Platform' dialog and fill in the platform name. For testing purposes `Mini` platform type is enough.
 
-Since we will be using Vagrant, please ensure you have Vagrant 1.6 or later installed. For more details, see the official [Vagrant installation docs](https://docs.vagrantup.com/v2/installation/index.html).
+<img src="_images/kontena-cloud-create-platform.png" alt="Create Platform" />
 
-After Vagrant is installed, you can install required plugins to Kontena CLI.
+After clicking 'Create' button Kontena Platform will be shortly provisioned to Kontena Cloud.
 
-```
-$ kontena plugin install cloud
-$ kontena plugin install vagrant
-```
+## Step 4. Install Kontena Nodes
 
-Before you can create a Kontena Cloud Platform you need to login:
+When the platform is ready you'll need some Kontena Nodes to run your containerized workloads.
 
-```
-$ kontena cloud login
-```
+After selecting Kontena Cloud as provider and clicking 'Add nodes' button, terminal windows should open and you can choose a node type and number of nodes you want to create.
```

Comment:
After selecting Kontena Cloud as the provider and clicking the 'Add nodes' button, terminal windows should open and you can choose a node type and the number of nodes you want to create.